WcaOnRails/app/webpacker/lib/requests/routes.js.erb

```diff
@@ -89,3 +89,5 @@ export const delegateProbationDataUrl = `<%= CGI.unescape(Rails.application.rout
 export const startDelegateProbationUrl = `<%= CGI.unescape(Rails.application.routes.url_helpers.start_delegate_probation_path) %>`;
 
 export const endDelegateProbationUrl = `<%= CGI.unescape(Rails.application.routes.url_helpers.end_delegate_probation_path) %>`;
+
+export const delegatesStaticPageDataUrl = `<%= CGI.unescape(Rails.application.routes.url_helpers.delegates_data_path(format: "json")) %>`;
```

Comment:
Is the `format: "json"` part strictly necessary? I don't recall ever using this for other routes...

Reply:
The reason was that the path had same URL as that of delegates page, and hence to distinguish format "json" was needed. I've now created a new scope under routes named `page_data` and added `get 'delegates'` there to get the data of delegates page. This helped to remove `format "json"`.

I could have actually named the function name in `static_controller` to something else like `delegate_data` to fix it, but I feel a new scope named `page_data` will be good to get the page data and maybe can be extended to fetching page data of other pages as well in future.